styles/buttons.css

```diff
@@ -0,0 +1,27 @@
+.button {
+  border-radius: 0.6rem;
+  padding: 1.2rem 2.4rem;
+  max-width: 24rem;
+  min-width: 22rem;
```

Comment:
why this min-width? what happens if the text is very short?

Reply:
If the text is very short, it should be at least that wide, i need